### PR TITLE
Darcyck hometown file changes

### PR DESCRIPTION
**Pull request by DarcyCK**
What do y'll think, @patrickmckenna ?

### CODE REVIEW

**Comment by patrickmckenna** [comment]
Given that description, think I wanna visit Billings, @DarcyCK!

File: docs/billings-mt.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Billings
+
+- The best thing about Billings is the view from the top of the Rimrocks, a sandstone bluff that runs along the north side of the Yellowstone valley.
+
+- You will drive up Zimmerman trail to the top of the Rims, and find a spot in one of the parks to pull over.
+- Sunsets are beautiful.
```
Comment (DarcyCK) [inline docs/billings-mt.md:6]: In highschool we used to have keggers on the Rims, but now our favorite bonfire/keg location is covered with mini-mansions

Comment (patrickmckenna) [inline docs/billings-mt.md:6]: Guessing the owners won't let you in just for nostalgia's sake?

File: docs/billings-mt.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+# Billings
+
+- The best thing about Billings is the view from the top of the Rimrocks, a sandstone bluff that runs along the north side of the Yellowstone valley.
```
Comment (spfortier) [inline docs/billings-mt.md:3]: What's the temperature there today?

File: docs/billings-mt.md (hunk 1 above)
Comment (DarcyCK) [inline docs/billings-mt.md:6]: Correct! I am sure it were some interesting archeological finds when digging for the foundation!

File: docs/billings-mt.md (hunk 2 above)
Comment (DarcyCK) [inline docs/billings-mt.md:3]: 54 F currently; for more info please see http://www.weather.com/weather/tenday/l/Billings+MT+59101:4:US

**Comment by spfortier** [comment]
Great job! @DarcyCK :+1:

**Pull request merged**